core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -226,7 +226,7 @@ trait WhiskActionsApi
                         val action = act.resolve(user.namespace)
                         onComplete(entitleReferencedEntities(user, Privilege.ACTIVATE, Some(action.exec))) {
                             case Success(true) =>
-                                transid.started(this, if (blocking) LoggingMarkers.CONTROLLER_ACTIVATION_BLOCKING else LoggingMarkers.CONTROLLER_ACTIVATION)
+                                transid.started(this, if (blocking) LoggingMarkers.CONTROLLER_ACTIVATION_BLOCKING else LoggingMarkers.CONTROLLER_ACTIVATION, s"${action.name}")
```

Comment:
Yes, we have the action name in marker invoker_activationRun_start. 
To handle also sequence actions in the perftool, I need to filter a marker by action name at the beginning of the chain in the controller (sequence names do not go to the invoker marker).

I will change to action.name.toString